zerolog/zerolog.go

```diff
@@ -0,0 +1,70 @@
+package zerolog
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/buger/jsonparser"
+	"github.com/rs/zerolog"
+)
+
+type Writer struct {
+	Gobrake *gobrake.Notifier
+}
+
+// New creates a new Writer
+func New(notifier *gobrake.Notifier) *Writer {
+	return &Writer{Gobrake: notifier}
+}
+
+// Write
+func (w *Writer) Write(data []byte) (int, error) {
+	lvlStr, err := jsonparser.GetUnsafeString(data, zerolog.LevelFieldName)
+	if err != nil {
+		return 0, fmt.Errorf("error getting zerolog level: %w", err)
+	}
+
+	lvl, err := zerolog.ParseLevel(lvlStr)
+	if err != nil {
+		return 0, fmt.Errorf("error parsing zerolog level: %w", err)
+	}
+
+	if lvl < zerolog.ErrorLevel || lvl > zerolog.PanicLevel {
+		return len(data), nil
+	}
+
+	var userData interface{}
+	err = json.Unmarshal(data, &userData)
+	if err != nil {
+		return 0, fmt.Errorf("error unmarshalling logs: %w", err)
```

Comment:
I'd rather not refer to our error notices as logs to avoid confusing the Airbrake user.

Reply:
This is not a notice here, currently at this point `data` is the error log that zerolog is about to send. So if we cant unmarshal it the log that the customer sent was malformed.

Reply:
Ok, that makes sense.